frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -332,20 +332,18 @@ function FileManagerMenu:setUpdateItemTable()
         })
     end
     if not Device.should_restrict_JIT then
+        local bb = require("ffi/blitbuffer")
```

Comment:
Naming the imported module `bb` is a bit confusing (although not that much here as it's used immediately) because `bb` is often used just for a buffer instance (`self.bb / local bb = Blitbuffer.new(...)`).
Best to stick with one the 2 variations we use everywhere:
```lua
local BlitBuffer = require("ffi/blitbuffer")
local Blitbuffer = require("ffi/blitbuffer")
```

Reply:
It's how you can tell apart vim person from emacs person without asking em what editor they use.